hdf5-vol-daos.spec

```
@@ -198,6 +199,9 @@ done
 %endif
 
 %changelog
+* Mon Feb 08 2021 Jonathan Martinez Montes <jonathan.martinez.montes@intel.com> - 1.1.0~rc2
```

You need to update the [`Release:`](https://github.com/daos-stack/hdf5-vol-daos/blob/2d51443677923ea1831c6bb8ffb7b6d8cd85cc8b/hdf5-vol-daos.spec#L40) and `%changelog` when you modify a specfile:

```suggestion
* Mon Feb 08 2021 Jonathan Martinez Montes <jonathan.martinez.montes@intel.com> - 1.1.0~rc2-2
```

Since the format on line 205 is also slightly incorrect, could you please fix it with:
```
* Wed Feb 03 2021 Maureen Jean <maureen.jean@intel.com> - 1.1.0~rc2-1
￼```